tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -124,124 +126,103 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 		}
 	})
 
-	ginkgo.Describe("[Basic]", func() {
-
-		// create three namespace
+	ginkgo.Describe("[Deploy]", func() {
 		ginkgo.BeforeEach(func() {
 			ns1 := namespaces[0]
 			namespaces = append(namespaces, ns1+"-1", ns1+"-2")
 		})
 
 		version := utilimage.TiDBLatest
-		clusterDomain := defaultClusterDomain
+		cluster1Domain := defaultClusterDomain
+		cluster2Domain := defaultClusterDomain
+		cluster3Domain := defaultClusterDomain
 
-		ginkgo.It("Deploy cluster across kubernetes", func() {
+		ginkgo.It("Deploy and delete cluster across kubernetes", func() {
 			ns1 := namespaces[0]
 			ns2 := namespaces[1]
 			ns3 := namespaces[2]
 
-			tc1 := GetTCForAcrossKubernetes(ns1, "basic-1", version, clusterDomain, nil)
-			tc2 := GetTCForAcrossKubernetes(ns2, "basic-2", version, clusterDomain, tc1)
-			tc3 := GetTCForAcrossKubernetes(ns3, "basic-3", version, clusterDomain, tc1)
+			tc1 := GetTCForAcrossKubernetes(ns1, "basic-1", version, cluster1Domain, nil)
+			tc2 := GetTCForAcrossKubernetes(ns2, "basic-2", version, cluster2Domain, tc1)
+			tc3 := GetTCForAcrossKubernetes(ns3, "basic-3", version, cluster3Domain, tc1)
 
 			ginkgo.By("Deploy the basic cluster-1")
-			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc1, 5*time.Minute, 10*time.Second)
+			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc1, 10*time.Minute, 10*time.Second)
 
 			ginkgo.By("Deploy the basic cluster-2")
-			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc2, 5*time.Minute, 10*time.Second)
+			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc2, 10*time.Minute, 10*time.Second)
 
 			ginkgo.By("Deploy the basic cluster-3")
-			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc3, 5*time.Minute, 10*time.Second)
+			utiltc.MustCreateTCWithComponentsReady(genericCli, oa, tc3, 10*time.Minute, 10*time.Second)
 
-			ginkgo.By("Deploy status of all clusters")
+			ginkgo.By("Check status of all clusters")
 			err := CheckClusterDomainEffect(cli, []*v1alpha1.TidbCluster{tc1, tc2, tc3})
 			framework.ExpectNoError(err, "failed to check status")
-		})
 
-		ginkgo.It("Deploy cluster with TLS-enabled across kubernetes", func() {
```

Comment:
Why delete this case?

Reply:
It has been reverted. This operation is for debugging.